lib/algo/sort/test.js

```diff
@@ -1,4 +1,6 @@
-const assert = require('chai').assert;
```

Comment:
Please take this out of this PR and raise a separate one. [Address only one use case in one PR.](https://github.com/sarbbottam/js-ds/blob/master/contributing.md#pull-request)